app/src/main/res/layout/profile_progress_header.xml

```diff
@@ -90,6 +90,7 @@
 
         <TextView
           android:id="@+id/ongoing_topics_count"
+          style="@style/TextViewCenter"
```

Comment:
Ditto here--this didn't have a center property before.

Reply:
To keep it consistent I added center property here. And by default it was center aligned based on the layout. Same with other files.

Reply:
Ah I see--that wasn't very clear. Thanks for clarifying.